src/commands.ts

```diff
@@ -525,3 +525,41 @@ export const openCmd = async (scriptId: any) => {
     process.exit(0);
   }
 };
+
+/**
+ * Lists available apis for the user.
+ */
+const apisList = async () => {
```

Comment:
Decrease the scope of this method by inlining it inside the `apis` method. You can keep it as a function, but inside that method.

```ts
export const apis = async () => {
  const list = async () => {};

```

This way,
- No other method in this file can use `apisList`. (No TS autocompletion suggestions either)
- We can simplify the name of the name of the method from `apisList` to `list` since `list` is part of the `apis` closure.

Reply:
Done.